projects/core/src/model/image.model.ts

```diff
@@ -15,6 +15,8 @@ export interface Image {
   galleryIndex?: number;
   imageType?: ImageType;
   url?: string;
+  width?: number;
+  height?: number;
```

Comment:
For a future newcomer, let's add JSDoc with more context, for those properties, explaining that those properties currently don't come from CMS, but were added allow for setting manually width & height of the images displayed by the MediaComponent.